docs/Getting Started/Activating the App/index.md

```diff
@@ -0,0 +1,7 @@
+## Activating the App
+
+At this point, the app should have started its SDL protocol session, registered through the RPC service, and be displayed in some sort of app list. The next step is to activate the app by selecting it from the list. This will inform the app that their HMI status has changed to one that is visible, and the app will start to send all its content to the head unit. 
```

Comment:
Don't agree with the change.

Reply:
The reason I made that change was that I thought "the list" was ambiguous, and because some apps may be activated at other times. e.g. if the head unit always wanted to start a particular app, or if it should be started in BACKGROUND (in which case the language I added is incorrect as well).

Reply:
I can't think of a better way to say this, will accept for now.